### ISSUE TITLE
Naming of Category

### ISSUE THREAD

**Issue by uranusjr** [stephenmcd/cartridge#104]
Hi,

I'm currently working on a recipe-sharing project. It has a similar model structure with Cartridge: A Displayable subclass for recipe, and a category page model to show a list of recipes. Cartridge works fine with it, of course, but I'm facing a little problem on admin customization — Cartridge's category model is named `Category` (with verbose names "Category/Categories"), and shows up in the Page admin's add menu as "Category," which to me sounds a bit too generic than it should. Of course, you can get what you want once you understands this, but still it might confuses people not familiar with Cartridge (and/or Mezzanine), which is not good since a CMS admin is supposed to be usable for non-programmers as well. Even programmers might need to spend some unneeded brain power trying to resolve between different kinds of categories if they have a more complicated structure.

So basically I'm purposing a change of name for model `Category`. It'd be good just changing the verbose names to Product category/categories. Refactoring model names would be more work, and you can always work around it by using `from cartridge.shop.models import Category as ProductCategory` if you need to (do you ever?).

Any thoughts?
